.github/workflows/test.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
+on:
+  pull_request:
+    branches: [ "master" ]
+jobs:
+  test:
+    name: test (JDK ${{ matrix.java }})
+    runs-on: ubuntu-latest
```

Comment:
Not sure if it's more stable to use fixed version of Ubuntu?

Reply:
AFAIK, many CI in GitHub actions uses `ubuntu-latest` as a default OS for Linux.